### PR TITLE
Update status pages

### PR DESCRIPTION
**Pull request by lumag**

### CODE REVIEW

**Comment by lumag** [comment]
@konradybcio-work could you please review?

File: _soc/x1e80100.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ status-crypto-rng: N
 status-debug-coresight:
 status-debug-dcc:
 status-debug-eud:
-status-display-dp: WIP
+status-display-dp: 6.9
```
Comment (konradybcio) [inline _soc/x1e80100.md]: does it actually work?

File: _soc/x1e80100.md [hunk 2]
```diff
@@ -81,12 +81,12 @@ status-storage-sdcc: N
...
-status-usb-hostmode: WIP
-status-usb-periphmode: WIP
-status-usb-typec: WIP
+status-usb: 6.9
+status-usb-hostmode: 6.9
+status-usb-periphmode: 6.9
+status-usb-typec: 6.9
 status-usb-usb4:
-status-usb-usbotg:
+status-usb-usbotg: 6.9
```
Comment (konradybcio) [inline _soc/x1e80100.md]: does it actually work?

File: _soc/x1e80100.md [hunk 3]
```diff
@@ -81,12 +81,12 @@ status-storage-sdcc: N
...
 status-storage-ufsice: N
 status-uart: 6.8-next
-status-usb: WIP
-status-usb-hostmode: WIP
-status-usb-periphmode: WIP
-status-usb-typec: WIP
+status-usb: 6.9
+status-usb-hostmode: 6.9
+status-usb-periphmode: 6.9
+status-usb-typec: 6.9
```
Comment (konradybcio) [inline _soc/x1e80100.md]: does it actually work?

File: _soc/x1e80100.md (hunk 1 above)
Comment (lumag) [reply _soc/x1e80100.md]: Fixed

**Pull request merged**